appveyor.yml

```diff
@@ -15,14 +15,13 @@ build:
   parallel: true
   verbosity: minimal
 deploy:
-  release: tgstation-server-3-v$(appveyor_build_version)
+  release: tgstation-server-v$(appveyor_build_version)
   description: 'The /tg/station server suite'
   provider: GitHub
   auth_token:
-    secure: github_oath_token
+    secure: 9Yf+FtoE7nf36V3dyVRANkNZ7og4dp/0uojCS51vrvIeNOTIM8Nq24lylgVBOhSL

```

Comment:
Appveyor is windows CI. The token is a repo scope github api token that's encrypted using a key stored in your appveyor account and decrypted on the build box. It only is given to builds that aren't from pull requests. And it's used for automatically pushing to github releases like so: https://github.com/Cyberboss/tgstation/releases/latest